code/game/objects/items/RCD.dm

```diff
@@ -633,6 +633,17 @@ RLD
 	matter = 500
 	canRturf = TRUE
 
+/obj/item/construction/rcd/bluespace
+	name = "bluespace RCD"
+	desc = "Compact, efficient bluespace RCD that is linked to the ORM,"
+	icon_state = "ircd"
+	sheetmultiplier = 4.4 //slightly better than original RCD due to tech advancement
+	w_class = WEIGHT_CLASS_SMALL
+	upgrade = RCD_UPGRADE_SILO_LINK
```

Comment:
Is there a method to link it to a new Ore Silo? I haven't looked at the silo link upgrade code recently.

Reply:
Yes there is, i ju-

F U C K.